brokernode-api/api/swagger.yaml

```diff
@@ -0,0 +1,303 @@
+---
+swagger: "2.0"
```

Comment:
This is the file I meant to comment on.  At some point we should try to use realistic examples like what you might see in the real application.

Reply:
yeh lets use real Tangle addresses / messages / trunk / branches